packages/cli/generators/repository/index.js

```diff
@@ -415,9 +415,11 @@ module.exports = class RepositoryGenerator extends ArtifactGenerator {
         type: 'list',
         name: 'repositoryBaseClass',
         message: PROMPT_BASE_REPOSITORY_CLASS,
-        when: this.artifactInfo.repositoryBaseClass === undefined,
         choices: availableRepositoryList,
-        default: availableRepositoryList[0],
+        default:
```

Comment:
@gczobel-f5 I feel the logic here has conflict with the code that [sets default value for option repositoryBaseClass](https://github.com/strongloop/loopback-next/blob/92e6f8063c5930acc7193c1a832fb542eec6d83e/packages/cli/generators/repository/index.js#L186-L190) and [assigns the options's value to artifactInfo](https://github.com/strongloop/loopback-next/blob/92e6f8063c5930acc7193c1a832fb542eec6d83e/packages/cli/generators/repository/index.js#L404-L422)
Like you said in https://github.com/strongloop/loopback-next/issues/2429, `this.artifactInfo.repositoryBaseClass` will never be undefined....

Reply:
You are right. I returned the default to the original option.